<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuView</class>
 <widget class="QWidget" name="MenuView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>meinHaushalt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>270</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Gear Rosemary</family>
             <pointsize>30</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
color: rgb(190, 190, 0)
}</string>
           </property>
           <property name="text">
            <string>@USER</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="infoButton">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color: white;
background-color: rgb(191, 187, 0); 
border-radius: 20px;
border-style: outset;
padding: 10px;
}

QPushButton:disabled {

}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(193, 189, 0)
);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(
cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd
);
}

QPushButton:focus {
	border: none;
	outline: none;
}</string>
           </property>
           <property name="text">
            <string>Daten ändern</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="passwordButton">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color: white;
background-color: rgb(191, 187, 0); 
border-radius: 20px;
border-style: outset;
padding: 10px;
}

QPushButton:disabled {

}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(193, 189, 0)
);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(
cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd
);
}

QPushButton:focus {
	border: none;
	outline: none;
}</string>
           </property>
           <property name="text">
            <string>Passwort ändern</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="payoptionButton">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color: white;
background-color: rgb(191, 187, 0); 
border-radius: 20px;
border-style: outset;
padding: 10px;
}

QPushButton:disabled {

}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(193, 189, 0)
);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(
cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd
);
}

QPushButton:focus {
	border: none;
	outline: none;
}</string>
           </property>
           <property name="text">
            <string>Zahlungsarten</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="topMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="adminToolsLabel">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Admin-Tools</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="createUserButton">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color: white;
background-color: rgb(191, 187, 0); 
border-radius: 20px;
border-style: outset;
padding: 10px;
}

QPushButton:disabled {

}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(193, 189, 0)
);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(
cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd
);
}

QPushButton:focus {
	border: none;
	outline: none;
}</string>
           </property>
           <property name="text">
            <string>Benutzer anlegen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="manageUserButton">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color: white;
background-color: rgb(191, 187, 0); 
border-radius: 20px;
border-style: outset;
padding: 10px;
}

QPushButton:disabled {

}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(193, 189, 0)
);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(
cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd
);
}

QPushButton:focus {
	border: none;
	outline: none;
}</string>
           </property>
           <property name="text">
            <string>Benutzer verwalten</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="categoryButton">
           <property name="minimumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>270</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color: white;
background-color: rgb(191, 187, 0); 
border-radius: 20px;
border-style: outset;
padding: 10px;
}

QPushButton:disabled {

}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(193, 189, 0)
);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(
cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd
);
}

QPushButton:focus {
	border: none;
	outline: none;
}</string>
           </property>
           <property name="text">
            <string>Kategorien</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>von</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="fromDate">
               <property name="minimumSize">
                <size>
                 <width>115</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>115</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="displayFormat">
                <string>dd.MM.yyyy</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>bis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="toDate">
               <property name="minimumSize">
                <size>
                 <width>115</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>115</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="displayFormat">
                <string>dd.MM.yyyy</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="categoryCheckBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="categoryComboBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="currentText">
              <string>&lt;Alle Kategorien&gt;</string>
             </property>
             <item>
              <property name="text">
               <string>&lt;Alle Kategorien&gt;</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="groupCheckBox">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Kategorien gruppieren</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
border-style: outset;
}</string>
             </property>
             <property name="placeholderText">
              <string>Durchsuchen</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="transactionsTable">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>125</number>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>30</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="DeleteButton">
               <property name="minimumSize">
                <size>
                 <width>270</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>270</width>
                 <height>45</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
color: white;
background-color: rgb(191, 187, 0); 
border-radius: 20px;
border-style: outset;
padding: 10px;
}

QPushButton:disabled {

}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(193, 189, 0)
);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(
cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd
);
}

QPushButton:focus {
	border: none;
	outline: none;
}</string>
               </property>
               <property name="text">
                <string>Ausgewählte löschen</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="transactionButton">
               <property name="minimumSize">
                <size>
                 <width>270</width>
                 <height>45</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>270</width>
                 <height>45</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
color: white;
background-color: rgb(191, 187, 0); 
border-radius: 20px;
border-style: outset;
padding: 10px;
}

QPushButton:disabled {

}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(193, 189, 0)
);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(
cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd
);
}

QPushButton:focus {
	border: none;
	outline: none;
}</string>
               </property>
               <property name="text">
                <string>Transaktion erstellen</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="saldoLabel">
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Saldo</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="saldoAmountLabel">
               <property name="minimumSize">
                <size>
                 <width>110</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>@saldo</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="logoutButton">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>45</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
color: rgb(197, 194, 0);
background-color: white; 
border-radius: 20px;
border-style: outset;
padding: 10px;
}

QPushButton:disabled {

}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 grey
);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(
cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd
);
}

QPushButton:focus {
	border: none;
	outline: none;
}</string>
         </property>
         <property name="text">
          <string>Abmelden</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
